invenio_app_rdm/config.py

```diff
@@ -112,6 +109,11 @@ def _(x):
 THEME_FRONTPAGE_TITLE = _('Invenio App RDM')
```

Comment:
Moderate: Shall we put here "The turn-key research data management repository"

Reply:
Actually, we should probably just have the cookiecutter title override this anyway. Made a PR for that: https://github.com/inveniosoftware/cookiecutter-invenio-rdm/pull/24

Reply:
Nice idea!